src/cli/src/error.rs

```diff
@@ -1,12 +1,13 @@
 use std::io::Error as IoError;
 
 use fluvio::FluvioError;
-use fluvio_extension_common::output::OutputError;
-use fluvio_extension_consumer::ConsumerError;
 use fluvio_cluster::cli::ClusterCliError;
-use crate::common::target::TargetError;
 use fluvio_sc_schema::ApiError;
 use fluvio_sc_schema::errors::ErrorCode;
+use fluvio_extension_common::output::OutputError;
+
+use crate::common::target::TargetError;
+use crate::consumer::error::ConsumerError;
```

Comment:
under features flag

Reply:
Hmmmmm..... you know I think it is a really very bad idea to `cfg` variants in a public enum. It causes this weird schism where the enum would be considered two different types under the two different feature flags... I think we should avoid that at all costs.

Reply:
Is there reason why this need explicit dependency to ConsumerConfig?